### ISSUE TITLE
[VOTE] Mark JSON-LD 1.0 as superseded

### ISSUE THREAD

**Issue by BigBlueHat** [w3c/json-ld-wg#154]
The Chairs and Editors propose that the [JSON-LD 1.0 specification](https://www.w3.org/TR/json-ld1/) should be marked as superseded by the [JSON-LD 1.1](https://www.w3.org/TR/json-ld/) specification released July 11, 2020.

The 1.1 specifications provide 1.0 compatibility. Additionally, processors can use the [`json-ld-1.0` processing mode](https://www.w3.org/TR/json-ld/#dfn-processing-mode) to explicitly ignore 1.1 features seen in data documents and contexts:
> The API provides an option for setting the processing mode to `json-ld-1.0`, which will prevent JSON-LD 1.1 features from being activated, or error if `@version` entry in a context is explicitly set to 1.1. This specification extends JSON-LD 1.0 via the json-ld-1.1 processing mode.

We believe this is sufficient to mark the 1.0 specification as superseded.

Please provide your vote as a comment on this issue.

Thanks!

**Comment by gkellogg** [comment]
👍

**Comment by davidlehn** [comment]
+1

**Comment by dlongley** [comment]
+1

**Comment by rubensworks** [comment]
👍

**Comment by pchampin** [comment]
+1

**Comment by BigBlueHat** [comment]
👍

**Comment by iherman** [comment]
I think we can call this. I will start the relevant administration.

**Comment by TallTed** [comment]
1.0 is marked as superseded, but it lacks the big red "this is old, go to the new" that have been added to more recent superseded docs, and which help a LOT in preventing people from discovering the new doc late in their process, having wasted time conforming to the old doc.

**Comment by gkellogg** [comment]
Maybe this is something that still needs attention from @pchampin? Reopening.

**Comment by pchampin** [comment]
"superceded" and the "outdated" red popup play slightly different roles.

* The red popup appears whenever a more recent version of the *same* specification exists. In particular, it appears when you click on the "Previous version" link in any spec.
* Superceded is a relationship between two *distincts* specifications belonging to the same *series*.

In the case of JSON-LD Syntax (the same applies to JSON-LD API), we have two specifications, namely JSON-LD 1.0 (see its [history](https://www.w3.org/standards/history/json-ld/)) and JSON-LD 1.1 (see its [history](https://www.w3.org/standards/history/json-ld11/)). In each case, the red popup appears if you click on any version but the last one. Both belong to the [json-ld](https://api.w3.org/specification-series/json-ld) series, and so the short URL [TR/json-ld](https://www.w3.org/TR/json-ld) points to the latest version of the latest specification of the series.

Other examples are [dcat-vocab-2](https://www.w3.org/TR/vocab-dcat-2/) superceding [dcat-vocab-1](https://www.w3.org/TR/vocab-dcat-1/), or [rdf11-concepts](https://www.w3.org/TR/rdf11-concepts/) superceding [rdf10-concepts](https://www.w3.org/TR/rdf10-concepts/) (although the latter does not use the "superceded" terminology for historical reasons). Note that the superceded versions do not display the red popup either. There might be counter examples elsewhere (this things have evolved organically, and our systeam manages a lot of special cases), but I believe the JSON-LD is the expected behaviour.

**Comment by TallTed** [comment]
> [@pchampin] I believe the JSON-LD is the expected behaviour

It is not *my* expected behavior, especially given that when you go to [JSON-LD 1.0](https://www.w3.org/TR/json-ld1/), it has a **Latest published version** of [http://www.w3.org/TR/json-ld/](https://www.w3.org/TR/json-ld/) — which is (today) JSON-LD 1.1, and will (soon-ish) be JSON-LD 1.2!

Also note that [JSON-LD 1.0](https://www.w3.org/TR/json-ld1/)'s **Previous version** is [http://www.w3.org/TR/2013/PR-json-ld-20131105/](https://www.w3.org/TR/2013/PR-json-ld-20131105/), which has the big red popup, which says, "For the latest version, please look at https://www.w3.org/TR/json-ld/" -- which, again, is JSON-LD 1.1!

Bottom line, if "the JSON-LD is the expected behaviour", I strongly believe those expectations (and that behavior) should be changed.

**Comment by pchampin** [comment]
> > [@pchampin] I believe the JSON-LD is the expected behaviour
> 
> It is not _my_ expected behavior, especially given that when you go to [JSON-LD 1.0](https://www.w3.org/TR/json-ld1/), it has a **Latest published version** of [http://www.w3.org/TR/json-ld/](https://www.w3.org/TR/json-ld/) — which is (today) JSON-LD 1.1, and will (soon-ish) be JSON-LD 1.2!
> 
> Also note that [JSON-LD 1.0](https://www.w3.org/TR/json-ld1/)'s **Previous version** is [http://www.w3.org/TR/2013/PR-json-ld-20131105/](https://www.w3.org/TR/2013/PR-json-ld-20131105/), which has the big red popup, which says, "For the latest version, please look at https://www.w3.org/TR/json-ld/" -- which, again, is JSON-LD 1.1!

You are right, this is confusing, and should be fixed. But I will argue that this does not contradict what I wrote above. Here is why:

* The short name of JSON-LD is (again for historical reasons), `json-ld`. So the short URL for this *specification* started as https://www.w3.org/TR/json-ld/.
* When JSON-LD 1.1 was started, a *series* was created, whose short name is `json-ld`, repurposing the above URL to be the tip of the series rather than the tip of the JSON-LD 1.0 specification. Anyway, both were identical until then, so this is [cool](https://www.w3.org/TR/cooluris/)).
* At the same time, an alias `json-ld1` was created to provide short URLs for the JSON-LD 1.0 specification.

So in the two cases you cite above, `https://www.w3.org/TR/json-ld/` should be replaced with `https://www.w3.org/TR/json-ld1/`, in order to be consistent with the rest.

> Bottom line, if "the JSON-LD is the expected behaviour", I strongly believe those expectations (and that behavior) should be changed.

I disagree that it should be changed, but I grant you that 1) we must carefully fix all the inconsistencies that make it confusing, and 2) we should probably add an entry in the superceded recommendation's header, counterpart of the "Previous recommendation" entry in the superceding recommendation's header.

**Comment by TallTed** [comment]
> 1) we must carefully fix all the inconsistencies that make it confusing, and 2) we should probably add an entry in the superceded recommendation's header, counterpart of the "Previous recommendation" entry in the superceding recommendation's header.

+1
